### PR TITLE
venv: add page

### PR DESCRIPTION
**Pull request by aaryan200**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

File: pages/common/venv.md
Comment (kbdharun) [inline pages/common/venv.md:3]:
```suggestion
> Create lightweight virtual environments in python.
```

File: pages/common/venv.md
Comment (kbdharun) [inline pages/common/venv.md:8]:
```suggestion
` python -m venv {{path/to/new_venv}}`
```
I think this would be better.

File: pages/common/venv.md
Comment (kbdharun) [inline pages/common/venv.md:12]:
```suggestion
`source {{path/to/new_venv}}/bin/activate`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

I also have a doubt, isn't `venv` a module rather than an independent utility?

File: pages/common/venv.md
Comment (aaryan200) [reply pages/common/venv.md:8]: Hi
I think `new_venv` can get confused with `venv`.
Can we keep it as `new_virtual_env` ?

Comment (kbdharun) [reply pages/common/venv.md:8]: Sure, I will request a review from other maintainers for a second opinion.

**Comment by aaryan200** [comment]
@kbdharun
Yes `venv` is a module which needs to be downloaded using `sudo apt-get install -y python3-venv`.

File: pages/common/venv.md
Comment (kbdharun) [reply pages/common/venv.md:8]: > Hi I think `new_venv` can get confused with `venv`. Can we keep it as `new_virtual_env` ?

```suggestion
` python -m venv {{path/to/virtual_environment}}`
```

File: pages/common/venv.md
Comment (kbdharun) [inline pages/common/venv.md:12]:
```suggestion
- Activate the virtual environment (Linux and Mac os):

`source {{path/to/virtual_environment}}/bin/activate`

- Activate the virtual environment (Windows):

`{{path\to\virtual_environment}}\Scripts\activate.bat`
```

**Comment by aaryan200** [comment]
@kbdharun 
Thanks for updates

**Review by kbdharun — APPROVED** [review]

File: pages/common/venv.md
Comment (realFranco) [inline pages/common/venv.md:10]: Hello, thank you for the contribution, I will add a suggestion:

```suggestion
- Activate the virtual environment (Linux and Mac OS):
```

**Review by realFranco — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good.

**Pull request merged**
